src/app/core/import/import-history/import-history.component.html

```diff
@@ -5,7 +5,10 @@ <h3 i18n>Previous Imports</h3>
     <mat-expansion-panel [expanded]="i === 0">
       <mat-expansion-panel-header>
         <mat-panel-title>
-          {{ item.date | date }}
+          <span class="file-info">
+            <fa-icon [icon]="'file-alt'" class="margin-right-small"></fa-icon>
+            {{ item.filename || item.config?.filename }}
```

Comment:
In what scenario would item.filename be empty, so that we are using item.config?.filename as fall back?